### PR TITLE
feat(PLAT-4): adding freud-icon lib listing in storybook

### PR DESCRIPTION
**Pull request by kayran**
Adding icon list view with search and clipboard copy:

https://user-images.githubusercontent.com/474481/231293641-0f078e31-424f-4b68-8392-25da729c547b.mp4

### CODE REVIEW

File: packages/web/stories/icons/icon-class/all-icons/all-icons.component.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  baseMessage: string = 'Copied to clipboard';
+  message: string = this.baseMessage;
+
+  searchText: string = '';
+
+  constructor(private clipboardService: ClipboardService) {}
+  // constructor(private clipboardService: ClipboardService, private messageService: FreudMessageService) {}
+
+  ngOnInit(): void {
+    console.log(iconNamesJson);
```
Comment (jeanbispo) [inline packages/web/stories/icons/icon-class/all-icons/all-icons.component.ts:24]: remove console.log

File: packages/web/stories/icons/icon-class/all-icons/all-icons.component.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      this.iconNames = this.fullIconNames.filter((iconName) => iconName.includes(this.searchText));
+    } else {
+      this.iconNames = iconNamesJson;
+    }
+  }
+
+
+  toast(iconName: string) {
+    this.message = `"${iconName}" ${this.baseMessage}!`
+    console.log(this.message);
```
Comment (jeanbispo) [inline packages/web/stories/icons/icon-class/all-icons/all-icons.component.ts:50]: remove console.log

File: packages/web/stories/icons/icon-class/all-icons/all-icons.component.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  iconNames:string[] = [];
+  fullIconNames:string[] = [];
+
+  baseMessage: string = 'Copied to clipboard';
+  message: string = this.baseMessage;
+
+  searchText: string = '';
+
+  constructor(private clipboardService: ClipboardService) {}
+  // constructor(private clipboardService: ClipboardService, private messageService: FreudMessageService) {}
```
Comment (jeanbispo) [inline packages/web/stories/icons/icon-class/all-icons/all-icons.component.ts:21]: remove commented line

**Review by jeanbispo — CHANGES_REQUESTED** [review]

File: packages/web/stories/icons/icon-class/all-icons/all-icons.component.ts (hunk 1 above)
Comment (kayran) [reply packages/web/stories/icons/icon-class/all-icons/all-icons.component.ts:24]:
```suggestion
```

File: packages/web/stories/icons/icon-class/all-icons/all-icons.component.ts (hunk 3 above)
Comment (kayran) [reply packages/web/stories/icons/icon-class/all-icons/all-icons.component.ts:21]:
```suggestion
```

File: packages/web/stories/icons/icon-class/all-icons/all-icons.component.ts (hunk 2 above)
Comment (kayran) [reply packages/web/stories/icons/icon-class/all-icons/all-icons.component.ts:48]:
```suggestion
```

**Review by jeanbispo — APPROVED** [review]

**Pull request merged**